ledger/participant-integration-api/src/main/resources/db/migration/postgres/V50__Append_only_schema.sql

```diff
@@ -0,0 +1,172 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+
+---------------------------------------------------------------------------------------------------
+-- V50: Append-only schema
+--
+-- This is a major redesign of the index database schema. Updates from the ReadService are
+-- now written into the append-only table participant_events, and the set of active contracts is
+-- reconstructed from the log of create and archive events.
+---------------------------------------------------------------------------------------------------
+
+-- This migration is work in progress. Data continuity is not implemented yet.
+-- Even though this migration is hidden behind a feature flag, we add safety checks
+-- that prevent it from running on a non-empty database.
+CREATE TABLE safety_check(
```

Comment:
please place a TODO here that this will be removed as soon as data migration efforts and schema reached production readiness.